utils/drop-database.js

```diff
@@ -0,0 +1,37 @@
+const mysql = require('mysql2');
+
+const path = require('path');
+
+require('dotenv').config({
+  path: path.join(__dirname, '../.env.test'),
+});
+
+const { DB_PASSWORD, DB_NAME, DB_USER, DB_HOST, DB_PORT } = process.env;
+
+const setUpDatabase = async () => {
+  try {
+    const connection = await mysql.createConnection({
+      host: DB_HOST,
+      user: DB_USER,
+      password: DB_PASSWORD,
+      port: DB_PORT,
+    });
+
+    connection.query(`DROP DATABASE ${DB_NAME}`, () => connection.end());
+  } catch (err) {
+    // if something goes wrong, console.log the error and the current environment variables
```

Comment:
Comments like this are not really helpful, they just add noise to the file. Comments should really be used to explain _why_ something is being done the way it is and not _how_ (ideally, the code itself should be clear enough to not need any further explanation).